wpeverest/everest-forms
# Fix - Flatpickr localization issue

### All Submissions:


### Changes proposed in this Pull Request:
Previously Flatpickr localization enqueue through cdn link. This PR solve this problem.

### How to test the changes in this Pull Request:

1. Create a form.
2. Drag and Drop date and time field and change the date localization in another language.
3.  Check whether localization is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Flatpickr localization issue.
